apps/site/pages/en/index.mdx

```diff
@@ -26,6 +26,22 @@ layout: home
         <br />
         <small className="!text-xs">for Node.js 18 and below</small>
       </Button>
+
+      <div className="flex flex-row gap-2 justify-center xs:mt-6">
+        <WithNodeRelease status="Current">
+          {({ release }) =>
+            <BadgeGroup as="span" size="small" badgeText={release.versionWithPrefix}>
+              <Link href={`/blog/release/${release.versionWithPrefix}`}>Node.js Current</Link>
+            </BadgeGroup>}
+        </WithNodeRelease>
+
+        <WithNodeRelease status="LTS">
+          {({ release }) =>
+            <BadgeGroup as="span" size="small" kind="warning" badgeText={release.versionWithPrefix}>
+              <Link href={`/blog/release/${release.versionWithPrefix}`}>Node.js LTS</Link>
```

Comment:
I suggest spelling out "Long Term Support" -- there are no other references to LTS on the landing page and usually initialisms should be written out in full on the first occurrence. 

```suggestion
              <Link href={`/blog/release/${release.versionWithPrefix}`}>Node.js Long Term Support</Link>
```

Reply:
@richardlau could you re-review :)

Reply:
Your input is more than welcome, @richardlau